symfony/apps/orangehrm/i18n/messages.zz_ZZ.xml

```diff
@@ -7725,8 +7725,8 @@
         <target>FR_Instructions for resetting your password have been sent to Email</target>
       </trans-unit>
       <trans-unit id="1932">
-        <source>Successfully reset your password.</source>
-        <target>FR_Successfully reset your password.</target>
+        <source>Successfully reset your password</source>
```

Comment:
@Damith88 Those are unnecessary noise commit to the repo .. I would recommend work on test branch next time